ui/page/collection/internal/collectionHeader/internal/collectionSubtitle/index.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+
+import { selectClaimForId } from 'redux/selectors/claims';
+import {
+  selectCollectionDescriptionForId,
+  selectCountForCollectionId,
+  selectCollectionHasEditsForId,
+  selectMyPublishedCollectionCountForId,
+} from 'redux/selectors/collections';
+
+import CollectionHeader from './view';
```

Comment:
Could clean up a bunch of `CollectionHeader` in the index files, but minor (there are an extension of `CollectionHeader` after all)